toqito/channel_metrics/completely_bounded_spectral_norm.py

```diff
@@ -0,0 +1,24 @@
+"Compute the completely bounded spectral norm of a quantum channel"
+from toqito.channel_ops import dual_channel
+from toqito.channel_metrics import completely_bounded_trace_norm
+def cb_spectral_norm(phi):
```

Comment:
To maintain convention with the rest of the project, I would opt to rename the function to `completely_bounded_spectral_norm`. It is a bit longer, but more explicit, and if the user wishes to condense the name on import, they can do that if they wish.